docs/background-services.rst

```diff
@@ -0,0 +1,62 @@
+App background services
+=======================
+
+F5 ACI ServiceCenter has a few background services and tasks running which ensure proper functioning of certain features such as Dynamic Endpoint Discovery. 
+
+View app background services status
+```````````````````````````````````
+1. Click the user menu in the top-right corner.
+
+2. Click **Settings** → **Status**.
+
+3. This displays a table named **Services** with the status of all the background services. The services displayed in this table are celeryd, celerybeat and redis. Please check details of these services below.
+
+
++--------------+-----------------------+--------------------------------------------------------------------------------------------------------------+
+| Service Name | Description           | Purpose                                                                                                      |
++--------------+-----------------------+--------------------------------------------------------------------------------------------------------------+
+| celeryd      | Task scheduler        | This service is responsible for Dynamic endpoint discovery processing and TEEM report sends to F5.           |
```

Comment:
"... and TEEM reports sent to F5."    (do you need to define what TEEM is and does, or is that already somewhere else?)